### PR TITLE
DAF-4115 - Implement iOS Control center

### PR DESCRIPTION
**Pull request by mvn-hienvu-hn**
## Problem
From the 2nd video played, the Control center overwrites info of current video with played videos.

Due to the videos which have the same `Datasource Type` will alway have the same `textureId`, so the old `setupUpdateListener` won't be dispose in `stopOtherUpdateListener` and the `Control center` keep showing the old infos while playing new video.

## Solution
- Stop all old `setupUpdateListener` before setup the new one.
- Dispose old `NotificationInfo` when setup a new `DataSource` to refresh the `Control center` and clear `thumbnail` in `_artworkImageDict`

## Addition done
- Remove the caller to the native side `seekTo` command and only keep the `eventSink("seek")`  command to sync logic with other `Control center`'s command and prevent the `seek` command got call two time.
- Keep old info and disable `seek` in `Control center` while the pip plan limited video is playing. ([flutter side implement](https://github.com/dwango-nfc/dwango-app-ch/pull/1664))

## Ticket

https://dw-ml-nfc.atlassian.net/browse/DAF-4115

## Spec

JP: https://docs.google.com/spreadsheets/d/1iXu4Vz9wYpT6fHE7oK_W4vyHP_-BGTvk7HTcETFJ_0I/edit#gid=260929825

VN: https://docs.google.com/spreadsheets/d/1XLwXVbNo_r21YE8Ir4mDQcyAa9CbKDWA/edit#gid=814456320

## Screenshot

- Before:

https://github.com/dwango-nfc/betterplayer/assets/100773699/5f2f4f78-c2fe-4191-b373-5b7e6d250b5e

- After: 

https://github.com/dwango-nfc/betterplayer/assets/100773699/874d0545-0807-403e-a098-c798b66d8887

### CODE REVIEW

File: ios/Classes/BetterPlayerPlugin.m [hunk 1]
```diff
@@ -276,6 +270,13 @@ - (void) stopOtherUpdateListener: (BetterPlayer*) player{
 
 }
 
+// Refactor this function when there is more than 1 offline video
+- (BOOL)isPipPlanLimitedVideo: (BetterPlayer*) player{
```
Comment (mvn-thanhluu-hn) [inline ios/Classes/BetterPlayerPlugin.m]: We should add another flag to `datasource` like `shouldClearPreviousNotificationInfo` for making clear condition to check this case

**Review by mvn-thanhluu-hn — COMMENTED** [review]
@mvn-hienvu-hn 
Thanks.

Sorry, please check my comment. 🙏

**Review by mvn-thanhluu-hn — APPROVED** [review]
LGTM.

**Comment by mvn-hienvu-hn** [comment]
@noboru-i @HiroshiOshiro 
Could you please review this PR 🙇.

**Review by noboru-i — APPROVED** [review]
I cannot understand perfectly.
But, I cannot find any comments.

LGTM.

**Pull request merged**